src/me/StevenLawson/TotalFreedomMod/TFM_PlayerRank.java

```diff
@@ -2,6 +2,7 @@
 
 import me.StevenLawson.TotalFreedomMod.Config.TFM_ConfigEntry;
 import static me.StevenLawson.TotalFreedomMod.TFM_Util.DEVELOPERS;
+import static me.StevenLawson.TotalFreedomMod.TFM_Util.ADMIN;
```

Comment:
Change this to:

import static me.StevenLawson.TotalFreedomMod.TFM_Util.ADMIN;

But be sure to change this:

public static final List<String> ADMIN = Arrays.asList("cowgomoooo12", "OliverDatGuy");

To:

public static final List<String> EXECUTIVE = Arrays.asList("cowgomoooo12", "OliverDatGuy");